python_modules/libraries/dagster-gcp/dagster_gcp/bigquery/resources.py

```diff
@@ -96,3 +101,23 @@ def bigquery_resource(context):
     bq_resource = BigQueryResource.from_resource_context(context)
     with bq_resource.get_client() as client:
         yield client
+
+
+def fetch_last_updated_timestamp(
```

Comment:
Is a single invocation equally performant to multiple calls? I know that with Snowflake it's possible to essentially do a batch to save on round trips, but maybe not possible with BigQuery?

Reply:
i don't have numbers on single invocation vs multiple calls. I think it's possible if we use raw sql here, I think it's just not possible with the pythonic api. Happy to switch back and just use a single sql query.

Reply:
I don't have an opinion on whether we should switch implementations right now, but if we think there's a decent chance that doing it in bulk will be faster, then I'd lean towards exposing a bulk API instead of a single-table one.